app/src/main/java/org/schabi/newpipe/util/SerializedUtils.java

```diff
@@ -0,0 +1,29 @@
+package org.schabi.newpipe.util;
+
+import androidx.annotation.NonNull;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+public final class SerializedUtils {
+    private SerializedUtils() {
+    }
+
+    @NonNull
+    public static <T extends Serializable> T clone(@NonNull final T item,
+                                                   @NonNull final Class<T> type
+    ) throws IOException, SecurityException, NullPointerException, ClassNotFoundException {
```

Comment:
```suggestion
    ) throws IOException, SecurityException, ClassNotFoundException {
```

Reply:
It was move from [SerializedCache.java](https://github.com/TeamNewPipe/NewPipe/pull/9410/files/298366a8eb815dbc662ef74252453954361e0d51#diff-02de746e84b7c32cf4ac88b48aa4cfc9b596e167ff75a5aa5679d8aa311b66a0), so I think we should not remove NullPointerException at here.

Reply:
I suggested the change as `NullPointerException` is a runtime exception and thus does not need to be explicitly declared in the `throws` list.